### PR TITLE
OTRS bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47529)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/OTRS/Integrations/OTRS/OTRS.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 SERVER = params.get('server', '').strip('/')
 USERNAME = params.get('credentials', {}).get('identifier')
 PASSWORD = params.get('credentials', {}).get('password')
+USE_LEGACY_SESSIONS = params.get('use_legacy_sessions', False)
```
Comment (adi88d) [inline Packs/OTRS/Integrations/OTRS/OTRS.py:17]:
```suggestion
USE_LEGACY_SESSIONS = argToBoolean(args.get('use_legacy_sessions', False))
```
```suggestion
USE_LEGACY_SESSIONS = params.get('use_legacy_sessions', False)
```

Comment (adi88d) [inline Packs/OTRS/Integrations/OTRS/OTRS.py:17]:
```suggestion
USE_LEGACY_SESSIONS = argToBoolean(args.get('use_legacy_sessions', False))
```

Comment (adi88d) [inline Packs/OTRS/Integrations/OTRS/OTRS.py:17]:
```suggestion
USE_LEGACY_SESSIONS = argToBoolean(args.get('use_legacy_sessions', False))
```

**Review by adi88d — CHANGES_REQUESTED** [review]
nice!

**Review by adi88d — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@adi88d @yaakovi Doc review completed.

**Comment by ShirleyDenkberg** [comment]
@adi88d @yaakovi I cannot edit the files.  
Here are the changes that are needed:
In the yml file, line 57, V8 should be v8.  (I think the v is always lower case).
Same comment for line 4 of the md file.

**Comment by dansterenson** [comment]
@ShirleyDenkberg The PR was merged already since the doc review was completed. In order to add changes to the file we need to open a new PR.

**Comment by ShirleyDenkberg** [comment]
@dansterenson OK.  The change is insignificant.  No reason to open another PR.

**Pull request merged**